amethyst_input/src/input_handler.rs

```diff
@@ -389,6 +393,43 @@ impl<T: BindingTypes> InputHandler<T> {
     pub fn send_frame_begin(&mut self) {
         self.mouse_wheel_vertical = 0.0;
         self.mouse_wheel_horizontal = 0.0;
+
+        // Prepare the mouse axis input
+        {
+            let hits_boundaries = |delta: f64| delta >= 1. || delta <= -1.;
```

Comment:
Why does this check for numbers greater than `1` and less than `-1`? It's only used on normalized values, so it's guaranteed to be in the range of `]-1; 1[`, isn't it?

Reply:
As the name implies, it looks for values on the boundary, so if the delta is `1.`, it returns `true`

Reply:
As the name implies, it looks for values on the boundary, so if the delta is `1.`, it returns `true`

Reply:
Yes, of course, I understand that. I'm merely asking why it checks for values _outside_ those limits. If the input values range only from `-1` to `1` inclusive, you could use simple equality instead.

```rust
let hits_boundaries = |delta: f64| delta == 1. || delta == -1.;
```

Or even using `abs` if you feel like it, although I suppose it's not worth the overhead to save 10 or so characters.

```rust
let hits_boundaries = |delta: f64| delta.abs() == 1.;
```

Reply:
> why it checks for values outside those limits

just a tick of mine. All these lines of code are equal, so if there is a performance difference, or a problem from a DX perspective, I am the last person to not do the switch to a better alternative 😉

Reply:
Well, I don't think it matters, really. The CPU will happily do the comparison all the same, so if you're happy with it as it is, feel free to leave it like that.